.buildkite/pipeline.verify.yml

```diff
@@ -0,0 +1,97 @@
+env:
+  PANTS_CONFIG_FILES: "['pants.toml', 'pants.ci.toml']"
+  BUILDKITE_PLUGIN_VAULT_ENV_SECRET_PREFIX: "secret/data/buildkite/env"
+
+steps:
+
+  - label: ":lint-roller::bash: Lint Bash"
+    command:
+      - make lint-bash
+    # plugins:
```

Comment:
You can probably delete the commented lines here

Reply:
Actually, I need to wire this repo up in Toolchain and Vault... these will then be the plugin configurations we need.